src/app/handlers/blacklist.handler.ts

```diff
@@ -1,9 +1,16 @@
 import { TextChannel } from 'discord.js';
 import Constants from '../../common/constants';
-import { IContainer, IHandler, IMessage, ClassType } from '../../common/types';
-
+import { IContainer, IHandler, IMessage, ClassType, linkLabel } from '../../common/types';
```

Comment:
Capitalise `linkLabel`. Also I doubt it'll be used outside this file (blacklist.handler), so might as well just define the type here.